hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.controller.js

```diff
@@ -269,6 +186,30 @@ define([
       return modalConfirmEdit.result;
     }
 
+    /**
+     * Confirms that the contract change is valid and saves it.
+     */
```

Comment:
1. Please add params description.
2. Can we make this func a promise?
3. This func should be renamed: validateContractAndSave()